sklearn/linear_model/_omp.py

```diff
@@ -1079,8 +1079,11 @@ def fit(self, X, y, **fit_params):
             Parameters to pass to the underlying splitter.
 
             .. versionadded:: 1.4
-                Only available if `enable_metadata_routing=True`. See
-                :ref:`Metadata Routing User Guide <metadata_routing>` for
+                Only available if `enable_metadata_routing=True` is set via
+                    :func:`~sklearn.set_config` or via
+                    :func:`~sklearn.config_context`.
```

Comment:
we shouldn't be mentioning the `config_context`, since it'll cause problems in this case if users don't use it right.

Reply:
fair enough.

Reply:
Oh my, I will remove this again then. :)